content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,133 @@
+# data status
+
+Show changes in the data tracked by DVC in the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc data status [-h] [-q | -v]
+                       [--granular] [--unchanged]
+                       [--untracked-files [{no,all}]]
+                       [--json]
+```
+
+## Description
+
+The `data status` command displays the state of the working directory and the
+committed changes with respect to the last Git commit (`HEAD`).
+
+It shows you what the new changes have been committed to DVC, which haven't been
+committed, which files aren't being tracked by DVC and Git, and what files are
+missing from the cache. It shows you what's been going on with `dvc add`,
+`dvc commit` and `dvc repro` commands.
+
+The `dvc data status` command only outputs information, it won't modify or
+change anything in your working directory. It's a good practice to check the
+state of your repository before doing `git commit` so that you don't
+accidentally commit something you don't mean to.
+
+An example output might look something like follows:
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to download files)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+```
+
+As shown above, the `dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
+  and `.dvc` files but the corresponding cache files are missing.
+- _DVC committed changes_ indicates that there are changes that are committed
+  that differs with the last Git commit. There might be more detailed state on
+  how each of those files changed: _added_, _modified_, and _deleted_.
+- _DVC uncommitted changes_ indicates that there are changes in the working
+  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
+  there might be more detailed state on how each of those files changed.
+- _Untracked files_ shows the files that are not being tracked by DVC and Git.
+  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
+  is specified.
+- _DVC Unchanged files_ shows the files that are not changed. This is not shown
+  by default, unless [`--unchanged`](#--unchanged) is specified.
```

Comment:
I would love to get rid of this and make them obvious. `git` for example never talks about these categories, they just assume maybe because `staged`/`unstaged` are a fundament concept in git.

Reply:
yes, you are right in the intention. But I would argue that Git is very complex as a product. You need to spend hours before you understand staged / unstaged. Only after that I would agree that it can start making sense w/o any explanations. In our case we would have to invest tons of time into UG, write DVC book, etc, etc